apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2/Detection.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Translate.v2.Data;
+
+namespace Google.Cloud.Translation.V2
+{
+    /// <summary>
+    /// One option as the result of detecting the language of a piece of text.
```

Comment:
This reads a little odd. I had to read it a few times and see where it was used to get it. Perhaps adding the word "possible" -> "One possible option..." might help? Also, maybe have a seealso link to `TranslationClient.DetectLanguage` (and async) below the summary.